### ISSUE TITLE
PySolFC on MacOS keystrokes work some of the time.

### ISSUE THREAD

**Issue by k-h** [shlomif/PySolFC#345]
I'm using PySolFC 2.20.1 on MacOS 10.15.7.  The keys only work sometimes.  Mostly PySolFC ignores keystrokes, but sometimes they work fine.

**Comment by k-h** [comment]
Whenever I hide or maybe unhide PysolFC it toggles the keystrokes on or off.  It seems to remember the current state of listening to keystrokes if I open it after  quitting.

**Comment by kingjon3377** [comment]
I've seen the same behavior; the workaround I've found to work is to Command-Tab away and then back.

(While verifying which version I have, Command-X---to save my current game and close the application---didn't work after I closed the "About" window until I switched to a different application and back.)

**Comment by k-h** [comment]
> I've seen the same behavior; the workaround I've found to work is to Command-Tab away and then back.
> 
> (While verifying which version I have, Command-X---to save my current game and close the application---didn't work after I closed the "About" window until I switched to a different application and back.)

Yeah, when it is not listening to keystrokes, it doesn't listen to any keystrokes.  None of the keystrokes listed in the menus work.  

And BTW, that hold and quit ⌘X option should be the default quit ⌘Q.  Whatever the current ⌘Q quit does, like exit and forget should be ⌘X.
